pkg/quarterdeck/db/models/apikeys.go

```diff
@@ -26,7 +26,7 @@ type APIKey struct {
 	Name        string
 	OrgID       ulid.ULID
 	ProjectID   ulid.ULID
-	CreatedBy   sql.NullByte
+	CreatedBy   ulid.ULID
```

Comment:
This is apparently 1 byte, not `[]byte` 😨 -- I just changed it to `ulid.ULID` to make things simpler RE our conversation in #99